### PR TITLE
birtreports: Use Log4j 2.x

### PR DESCRIPTION
**Pull request by pranavsaxena17**
Update build file, CHANGELOG, and change code to use updated logging infrastructure.
Part of: zaproxy/zaproxy#6376

### CODE REVIEW

File: addOns/birtreports/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed
```
Comment (thc202) [inline addOns/birtreports/CHANGELOG.md:7]: There's already a changed section.

Comment (pranavsaxena17) [reply addOns/birtreports/CHANGELOG.md]: Done

File: addOns/birtreports/CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,14 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
```
Comment (thc202) [inline addOns/birtreports/CHANGELOG.md:7]: Revert.

File: addOns/birtreports/CHANGELOG.md [hunk 3]
```diff
@@ -4,12 +4,14 @@ All notable changes to this add-on will be documented in this file.
...
 ### Added
 - Add help.
 - Add info and repo URLs.
 
 ### Changed
-- Update minimum ZAP version to 2.9.0.
+- Update minimum ZAP version to 2.10.0.
+- Now using 2.10 logging infrastructure (Log4j 2.x).
```
Comment (thc202) [inline addOns/birtreports/CHANGELOG.md:14]: This is not really needed, given the above.

**Comment by thc202** [comment]
The commits can be squashed (better amend when addressing comments).

**Comment by pranavsaxena17** [comment]
@thc202 Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Comment by pranavsaxena17** [comment]
Thanks for reviewing! :smile:

**Comment by kingthorin** [comment]
@pranavsaxena17 please let us know how you'd like to be listed in the credits (have a look at others and let us know what name/handle/twitter account):
https://www.zaproxy.org/docs/desktop/credits/#zap-extended-team

**Comment by pranavsaxena17** [comment]
@kingthorin Just my name ("Pranav Saxena") and my github handle would be nice :smiley:

**Pull request merged**